contribute/design/openebs-provisioner-high-availibility.md

```diff
@@ -1,44 +1,45 @@
 # High Availability for OpenEBS Volume Provisioner
-
+---
 ## Problems addressed:
 
 - `openebs-provisioner` should always be available.
-- `openebs-provisioner` should always be running on the specified/labeled nodes. 
+- `openebs-provisioner` should always be running on the specified/labeled pods. 
 - If any node in cluster goes down in that scenario the pod should be assigned to other node in the cluster.
- 
+---
 ## Solutions for above problems: 
 
 
-- Increase the `replicas` to 2 or >2 in `openebs-operator.yaml` for `openebs-provisioner`. 
+- Increase the `replicas` to 2 or >2 in `openebs-operator.yaml` for `openebs-provisioner` deployment. 
 
 
 - We need to make sure that each node is running at least one  `openebs-provisioner` pod.  
-  We need to label nodes to assign the pods on those. 
+  We need to label pods so that pods with specific labels will always be running on different nodes. 
 
   To do that execute: 
 
 ```
-kubectl label nodes kubeminion-01  node=minion01  # For node kubeminion-01
-kubectl label nodes kubeminion-02  node=minion02   # For node kubeminion-02
+kubectl label pods openebs-provisioner-1149663462-6vbvm  provisioner=P2
```

Comment:
1/ These steps are not required as we can set these labels in the provisioner specs directly.
2/ Let us quickly debate on the label key & label value in maya channel.
3/ Same k:v labels can be assigned into these Pods. This label(s) will repel the Pods.